google/cloud/bigtable/admin_v2/overlay/__init__.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2025 Google LLC.
```

Comment:
I'm a bit confused about the gapic folders in the file structure:

```
google
    cloud
        bigtable
            data
            admin
            admin_v2
        bigtable_v2
```

I remember some discussions around this before, but it still seems a bit awkward. Has this been locked in?